### ISSUE TITLE
[v2] GraphQL Configuration Scan invoked many times

### ISSUE THREAD

**Issue by RafaelKr** [jimkyndemeyer/js-graphql-intellij-plugin#230]
## Version and Environment Details
__Operation system__:
Linux Mint 19.1 Cinnamon 64-bit

__IDE name and version__:
```
IntelliJ IDEA 2019.1 (Ultimate Edition)
Build #IU-191.6183.87, built on March 27, 2019
Licensed to IntelliJ IDEA Evaluator
Expiration date: April 28, 2019
JRE: 1.8.0_202-release-1483-b39 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.15.0-47-generic

```
__Plugin version__:
v2.0.0-beta-2 (installed via Plugin Repository)

## Expected Behaviour
GraphQL Configuration Scan invocation should be debounced, so it only executes once.

## Actual Behaviour
GraphQL Configuration Scan seems to be invoked many times.
![image](https://user-images.githubusercontent.com/14234815/55635236-82204100-57c0-11e9-8b36-da6402f06e66.png)

**Comment by jimkyndemeyer** [comment]
Hi Rafael.

Thanks for testing the plugin.

Can you describe the steps needed to reproduce this? I'd like to understand the problem fully before implementing a fix.

Best regards,
Jim.

**Comment by RafaelKr** [comment]
Hey Jim,

in my case it is an Android project, and I don't event use graphql there (yet). I just ran a gradle sync.
I could imagine that there are many files updated, which triggers the Config Scan and instead of checking if there is already a scan requested it just puts it into the Background Task queue. But you're the specialist ;)

Thanks for your incredibly fast response! Have a nice weekend
Rafael

**Comment by jimkyndemeyer** [comment]
Fixed in the upcoming 2.0.0 release.
